olive/engine/footprint.py

```diff
@@ -422,3 +429,30 @@ def _plot_pareto_frontier(self, ranks=None, save_path=None, is_show=True, save_f
 
         if is_show:
             fig.show()
+
+
+def get_best_candidate_node(
```

Comment:
could you add some comments here to explain the logic?

Reply:
will do